config/routes.rb

```diff
@@ -9,8 +9,9 @@
     resources :events, only: :index do
       post :create_rsvp, on: :collection
     end
+    get '/people/targets',                 to: 'people#targets'
   end
-  post  '/calls/start',                    to: 'calls#start'
+  post '/calls/start',                    to: 'calls#start'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.